src/YarpUtilities/src/VectorsCollectionServer.cpp

```diff
@@ -96,6 +96,9 @@ bool VectorsCollectionServer::populateMetadata(const std::string& key,
                                                const std::vector<std::string>& metadata)
 {
     // check if the metadata has been already populated
+    // This code was temporarily removed via commenting it out, this will be discussed with the team
+    // later on how to append metadata after it is finalized
+
```

Comment:
Leftover

Reply:
Fixed in [ee4cf0d](https://github.com/ami-iit/bipedal-locomotion-framework/commit/ee4cf0dd55ef916922a4013375d9fc9014d13ece)